app/RockTheVoteRecord.php

```diff
@@ -163,21 +163,23 @@ private function parseVoterRegistrationStatus($rtvStatus, $rtvFinishWithState)
      */
     private function parsePostDetails($record)
     {
-        $details = [];
-
-        $importantKeys = [
-            'Tracking Source',
-            'Started registration',
-            'Finish with State',
-            'Status',
-            'Pre-Registered',
-            'Home zip code',
-        ];
+        $config = ImportType::getConfig(ImportType::$rockTheVote);
```

Comment:
I've seen this in a few spots now and it seems really confusing. Upon further inspection it looks like it's just grabbing the config for Rock The Vote. Why can't we just use the ole `config()` helper which is pretty straightforward? This series of static properties and methods just seems confusing to me. This can essentially be just:

```php
$config = config('import.rock_the_vote');
```